synapse/cores/lmdb.py

```diff
@@ -515,7 +640,9 @@ def _getSizeByProp(self, prop, valu=None, limit=None, mintime=None, maxtime=None
 
     def _getRowsByProp(self, prop, valu=None, limit=None, mintime=None, maxtime=None,
                        do_count_only=False):
-        indx = self.index_pt if valu is None else self.index_pvt
+        indx = self._dbs.get(PROP_VAL_INDEX)
```

Comment:
This is where the _dbs refactoring is going to cause a decent performance hit. Before, in order to get the db variable, there was 1 dict lookup. Now there are 4: `PROP_VAL_INDEX` local lookup, `PROP_VAL_INDEX` global lookup, lookup of '_dbs' into self, lookup of string into `dbs` dict. 3 extra dict lookups might not sound like a lot, but this function was < 10 microseconds before.

Note that synapse doesn't use constants ID=0, PROPS=1 for I think the same performance reason.